app/models/export/anything.rb

```diff
@@ -44,25 +38,14 @@ def model_plural
     end
 
     def output_io
-      case output_name
-      when :tmp
-        STDERR.puts("Exporting #{model_plural} to #{filename}")
-        File.open(filename, 'w+') do |file|
-          yield file
-        end
-      when :stdout
-        yield STDOUT
-      else
-        STDERR.puts usage
+      STDERR.puts("Exporting #{model_plural} to #{filename}")
+      File.open(filename, 'w+') do |file|
```

Comment:
Yay! Does this mean the specs will stop churning stuff out to the console too?

Reply:
👍

Reply:
oh, you say it does in the commit message. 🎉